### ISSUE TITLE
Undesired warnings when using PHP 7.1 nullable types

### ISSUE THREAD

**Issue by JeroenDeDauw** [squizlabs/PHP_CodeSniffer#1292]
I had to

```xml
    <exclude name="Squiz.WhiteSpace.OperatorSpacing.NoSpaceBefore" />
    <exclude name="Squiz.WhiteSpace.OperatorSpacing.NoSpaceAfter" />
```

Cause these sniffs complain about `returnStringOrNull(): ?string {`

The `?` is not used as an operator here, so the sniff should not apply.

    version 2.7.1 (stable)

**Comment by JeroenDeDauw** [comment]
Issue also present with `version 3.0.0RC2 (beta)`

**Comment by gsherwood** [comment]
This was fixed by commit https://github.com/squizlabs/PHP_CodeSniffer/commit/ee3ce4b8ab6b5c815a06fea52ab480a69343134e and will be released in the next version.

**Comment by JeroenDeDauw** [comment]
Awesome thanks! Any ETA?

**Comment by gsherwood** [comment]
> Awesome thanks! Any ETA?

Not at this stage. At least a few weeks out, but tends to depend on my day job.
